samples/sensor/stm32_temp_sensor/boards/stm32l562e_dk.overlay

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2022 Wouter Cappelle
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Application overlay for creating temperature sensor device instance
+ */
+
+/ {
+    stm-temp {
+        label = "DIE_TEMP";
+        compatible = "st,stm32-temp-cal";
+        io-channels = <&adc1 17>;
+        status = "okay";
+        ts-cal1-addr = <0x0BFA05A8>;
+        ts-cal2-addr = <0x0BFA05CA>;
+        ts-cal1-temp = <30>;
+        ts-cal2-temp = <110>;
+        ts-cal-vrefanalog = <3000>;
+        ts-cal-offset = <30>;
+    };
```

Comment:
Let's move this node directly in stm32l5.dtsi (with `status = "okay";`) . It will then be accessible to any other application.
In this file, you'll only need to change status node.

Reply:
good point! 
I think you meant `status = "disabled";` right?